platforms/core-runtime/native/src/main/java/org/gradle/internal/nativeintegration/services/NativeServices.java

```diff
@@ -108,7 +112,7 @@ public boolean initialize(File nativeBaseDir, boolean useNativeIntegrations) {
                         }
                     }
                     try {
-                        FileEvents.init(nativeBaseDir);
+                        fileEvents = FileEvents.init(nativeBaseDir);
```

Comment:
❓  Could we avoid having `NativeServices.fileEvents` as static? Looks a bit weird, since it's hard to know where `initialize` it's called from (it's even a public method :O). 
Would it make sense to pass  `NativeServices` in `initialize` and FILE_SYSTEM_WATCHING could set `NativeServicesfileEvents` there? And that way we could avoid having `static FileEvents fileEvents`.
WDYT?